### DISCUSSION TITLE
Research help for My website

### DISCUSSION THREAD

**Discussion by harryjason23** [community/community#42204] [New to GitHub]
### Select Topic Area

Product Feedback

### Body

Hey there i am new in Git hub I need help with some research for one part of the plot. In the story, a group take over the Snorkeling sand use an AI unsupervised machine learning algorithm  kindly fix my issue for my  [Snorkeling Gears And Water Gadgets](https://watersnorkeling.com/know-about-scuba-diving/)

**Comment by ghostinhershell** [comment]
Hey there! :wave:

Thanks for posting in the GitHub Community, @harryjason23 ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

Furthermore, you are more likely to get a useful response if you are posting your question(s) in the applicable category and are explicit about what your project entails--giving a few more details might help someone give you a nudge in the right direction.

I've gone ahead and moved this to the correct category for you. Good luck!
